opal/mca/memory/linux/memory_linux_component.c

```diff
@@ -162,6 +187,53 @@ static int linux_register(void)
                                             OPAL_INFO_LVL_3,
                                             MCA_BASE_VAR_SCOPE_READONLY,
                                             &opal_memory_linux_disable);
+    if (0 > ret) {
+        return ret;
+    }
+
+#if MEMORY_LINUX_MALLOC_ALIGN_ENABLED
+    mca_memory_linux_component.use_memalign = -1;
+    ret = mca_base_component_var_register(&mca_memory_linux_component.super.memoryc_version,
+                                 "memalign",
+                                 "[64 | 32 | 0] - Enable memory alignment for all malloc calls (default: disabled).",
```

Comment:
seems @ggouaillardet solved issue in master. Thanks.

only reason is to minimize changes with original code. Moreover using enumeration requires additional variable.